Example/Tests/Classes/LeanplumTest.m

```diff
@@ -1137,28 +1137,28 @@ - (void)test_variables
-    XCTAssertEqual(10, [[LPVarCache getVariable:@"int"] intValue]);
-    XCTAssertEqual(10.0f, [[LPVarCache getVariable:@"float"] floatValue]);
-    XCTAssertEqual(10.0, [[LPVarCache getVariable:@"double"] doubleValue]);
-    XCTAssertEqual(YES, [[LPVarCache getVariable:@"bool"] boolValue]);
-    XCTAssertEqual(10L, [[LPVarCache getVariable:@"long"] longValue]);
-    XCTAssertEqual(@10, [[LPVarCache getVariable:@"number"] numberValue]);
-    XCTAssertEqual(5.0f, [[LPVarCache getVariable:@"cgfloat"] cgFloatValue]);
-    XCTAssertEqual('c', [[LPVarCache getVariable:@"char"] charValue]);
-    XCTAssertEqual(100LL, [[LPVarCache getVariable:@"long_long"] longLongValue]);
-    XCTAssertEqual(0x1, [[LPVarCache getVariable:@"short"] shortValue]);
-    XCTAssertEqual(5, [[LPVarCache getVariable:@"unsigned_int"] unsignedIntValue]);
-    XCTAssertEqual(NSUIntegerMax, [[LPVarCache getVariable:@"unsigned_integer"]
+    XCTAssertNotNil([[LPVarCache sharedCache] getVariable:@"named"]);
+    XCTAssertEqual(10, [[[LPVarCache sharedCache] getVariable:@"int"] intValue]);
+    XCTAssertEqual(10.0f, [[[LPVarCache sharedCache] getVariable:@"float"] floatValue]);
+    XCTAssertEqual(10.0, [[[LPVarCache sharedCache] getVariable:@"double"] doubleValue]);
+    XCTAssertEqual(YES, [[[LPVarCache sharedCache] getVariable:@"bool"] boolValue]);
+    XCTAssertEqual(10L, [[[LPVarCache sharedCache] getVariable:@"long"] longValue]);
+    XCTAssertEqual(@10, [[[LPVarCache sharedCache] getVariable:@"number"] numberValue]);
+    XCTAssertEqual(5.0f, [[[LPVarCache sharedCache] getVariable:@"cgfloat"] cgFloatValue]);
+    XCTAssertEqual('c', [[[LPVarCache sharedCache] getVariable:@"char"] charValue]);
+    XCTAssertEqual(100LL, [[[LPVarCache sharedCache] getVariable:@"long_long"] longLongValue]);
+    XCTAssertEqual(0x1, [[[LPVarCache sharedCache] getVariable:@"short"] shortValue]);
+    XCTAssertEqual(5, [[[LPVarCache sharedCache] getVariable:@"unsigned_int"] unsignedIntValue]);
+    XCTAssertEqual(NSUIntegerMax, [[[LPVarCache sharedCache] getVariable:@"unsigned_integer"]
                                    unsignedIntegerValue]);
-    XCTAssertEqual(200, [[LPVarCache getVariable:@"unsigned_char"] unsignedCharValue]);
-    XCTAssertEqual(50L, [[LPVarCache getVariable:@"unsigned_long"] unsignedLongValue]);
-    XCTAssertEqual(25LL, [[LPVarCache getVariable:@"unsigned_long_long"] unsignedLongLongValue]);
-    XCTAssertEqual(0x1, [[LPVarCache getVariable:@"unsigned_short"] unsignedShortValue]);
-    XCTAssertEqual(1, [[LPVarCache getVariable:@"A"] integerValue]);
-    XCTAssertEqual(2, [[[LPVarCache getVariable:@"C"] objectForKey:@"a"] integerValue]);
+    XCTAssertEqual(200, [[[LPVarCache sharedCache] getVariable:@"unsigned_char"] unsignedCharValue]);
+    XCTAssertEqual(50L, [[[LPVarCache sharedCache] getVariable:@"unsigned_long"] unsignedLongValue]);
+    XCTAssertEqual(25LL, [[[LPVarCache sharedCache] getVariable:@"unsigned_long_long"] unsignedLongLongValue]);
```

Comment:
These lines are too long. You can extract `[LPVarCache sharedCache] ` to greatly simplify these, possibly more.